modules/http/src/main/java/org/mule/module/http/internal/listener/HttpMessageProcessorTemplate.java

```diff
@@ -48,6 +49,7 @@
     private HttpResponseBuilder responseBuilder;
     private HttpResponseBuilder errorResponseBuilder;
     private HttpThrottlingHeadersMapBuilder httpThrottlingHeadersMapBuilder = new HttpThrottlingHeadersMapBuilder();
+    private Map<String, String> extraHeaders = new HashMap<String, String>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2293))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7625&issues=AWiOv-8gotIXnmsU1tFt&open=AWiOv-8gotIXnmsU1tFt)

[ISSUE_KEY]: AWiOv-8gotIXnmsU1tFt